boards/adafruit/itsybitsy/doc/index.rst

```diff
@@ -123,7 +123,7 @@ The :zephyr:code-sample:`button` sample lets you test the buttons (switches) and
 The :zephyr:code-sample:`blinky` sample lets you test the red LED.
 
 The DotStar LED has been implemented as a SPI device and can be tested
-with the :zephyr:code-sample:`led-apa102` sample application.
```

Comment:
I am not sure but maybe we should mention in the release not that theses samples have been removed and replaced by the `led_strip` sample ?

Reply:
At a glance, It seems the removing samples are not usually mentioned in the release note.

Reply:
Not been mentioned before

Reply:
Perfect then !